machinelearning/tensorflow.md

```diff
@@ -300,11 +305,28 @@ docker run -p 8501:8501 --name tfserving_resnet --mount type=bind,source=/tmp/re
 
 While the Grappler optimizer covers majority of the networks, there are few scenarios where either the Grappler optimizer can't optimize the generic graph or the runtime kernel launch overhead is simply not acceptable. XLA addresses these gaps by providing an alternative mode of running models: it compiles the TensorFlow graph into a sequence of computation kernels generated specifically for the given model. TensorFlow-2.9.0 supports aarch64 xla backend with Eigen runtime. For the best performance please cherrypick the [PR](https://github.com/tensorflow/tensorflow/pull/55534) and rebuild the TensorFlow libraries.
 ```
+# Install gcc-10 and g++-10 as it is required for Arm Compute Library build
```

Comment:
for what ubuntu 20.04? should probably call that out since 22.04 is around and has a different default

Reply:
thanks, yes, this is for Ubuntu 20.04, will add a note.